app/views/shared/synchronizer.html.erb

```diff
@@ -9,29 +9,26 @@ end
 
 <% if media_url.present? %>
   <% # Note: "text-align:left;" is required for the widget container, otherwise the seek bar won't render properly %>
-  <div id="synchronizer-widget" style="text-align:left;" data-media-url="<%= media_url %>" data-<%= synch_mode %>-url="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: synch_mode) %>">
-    <div id="working-area" class="row" style="position:relative;">
-      <div id="left-sidebar" class="col-md-6">
-        <div id="media-playback" class="">
-          <div id="ytplayer"></div>
-          <div id="video"><video id="video-player" data-able-player data-include-transcript="false" preload="auto"><track id="video-captions" label="English" kind="captions" srclang="en" src="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: 'captions')%>" /></video></div>
-          <div id="audio"><audio id="audio-player" data-able-player data-include-transcript="false" preload="auto"><track id="audio-captions" label="English" kind="captions" srclang="en" src="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: 'captions')%>" /></audio></div>
+  <div id="synchronizer-widget" style="text-align:left;" data-media-url="<%= media_url %>" data-<%= synch_mode %>-url="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: synch_mode) %>" class="">
+    <div id="working-area" class="d-flex flex-column flex-md-row">
```

Comment:
to fix overflow and gap, change class to: class="d-flex flex-column flex-md-row mx-lg-n2"